chef/cookbooks/crowbar/recipes/crowbar-db-dump.rb

```diff
@@ -83,6 +83,10 @@
   # Stop remaining pacemaker resources
   bash "stop pacemaker resources" do
     code <<-EOF
+      crm stop postgresql
+      # HA: Deletes postgres data directory after stopping postgresql service
+      # but before stopping fs-postgresql.
+      rm -rf /var/lib/pgsql/data
```

Comment:
It does not hurt, does it? There might be some check, for example `not_if { ::File.exist?("/usr/sbin/crm") }` but it's not really necessary